python/lsst/ip/isr/assembleCcdTask.py

```diff
@@ -195,13 +197,15 @@ def assembleCcd(self, assembleInput):
         ccd = None
         if hasattr(assembleInput, "has_key"):
             # Get a detector object for this set of amps
-            ccd = assembleInput.itervalues().next().getDetector()
+            ccd = assembleInput.values().next().getDetector()
```

Comment:
Ideally all the `next()` methods would be replaced with `__next__()` in our python/C++ layer so that we can use the `next()` function where appropriate.